### DISCUSSION TITLE
Were do we report bugs in extensions API?

### DISCUSSION THREAD

**Discussion by vanowm** [microsoft/MicrosoftEdge-Extensions#127] [General]
The name of this repository is misleading, since it's only about webstore and partner center.
So, were do we report issues with extensions API?

**Comment by Bhuvana-Priya** [comment]
Hi, @vanowm ,

You may report the bugs regarding extensions API here in this repository or you may reach out to us via ext_dev_support@microsoft.com DL.

Thanks,
Bhuvana
